.github/workflows/upload-docs-to-aws.yml

```diff
@@ -0,0 +1,63 @@
+name: Upload Docs to S3
+
+on:
+  push:
+    branches:
+      - devnet
+      - testnet
+
+jobs:
+  build-and-upload-docs:
+    runs-on: ubuntu-latest
+    env:
+      S3_BUCKET: TODO
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      # ----------- Build Rust Docs -----------
+      - name: Install Rust stable
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
Should this use the rust version in `rust-toolchain.toml` instead?

Reply:
I think you can only hardcode it 🤔

Reply:
I wanted to check how the others in the repo tho it but then remembered that we use self hosted everywhere which has rust pre-installed 😆

Reply:
Isn't needed at all because the GitHub action image already contains rust